src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Engine/TreeData.StructuredTrivia.cs

```diff
@@ -17,13 +19,13 @@ private class StructuredTrivia : TreeData
             private readonly TreeData _treeData;
 
             public StructuredTrivia(SyntaxTrivia trivia, int initialColumn)
-                : base(trivia.GetStructure())
+                : base(trivia.GetStructure()!)
             {
                 Contract.ThrowIfFalse(trivia.HasStructure);
 
                 _trivia = trivia;
 
-                var root = trivia.GetStructure();
+                var root = trivia.GetStructure()!;
```

Comment:
aside.  seems strange that this is passed both into the base class, and reacquired here.

Reply:
Yes, I left this though.